lib/doctest.js

```diff
@@ -144,12 +144,12 @@ const rewriteJs = sourceType => ({
       if (type === 'Line') {
         maybePushLine (value, start, loc.start.line);
       } else {
-        let offset = start;
-        let number = loc.start.line;
-        for (const text of value.split ('\n')) {
-          maybePushLine (text.replace (/^\s*[*]/, ''), offset, number);
-          offset += '\n'.length;
```

Comment:
This should have been `offset += text.length + '\n'.length;` (it was not actually a problem since offsets are only used for ordering).